### DISCUSSION TITLE
New _hsr Keyword

### DISCUSSION THREAD

**Discussion by djbbiz** [cortexrd/Knack-Toolkit-Library#248] [General]
It may be me, but I am confused as to what this keyword does and where it does it.

_hsr=[dr, duration, rate], [clr, outlineColor], [cls, class1, class2, …], [style, styleString]
Highlight Submitted Record

Used in: Views

My assumption was if you add the keyword to a form it would highlight the last record submitted in a Grid View for example. But that doesn't seem to be the case for me. It says the first parameter is for the confirmation message, but  don't see any change there either but not sure its use there. Can you elaborate or show a image of what this does and where it does it? Thank you.

**Comment by cortexrd** [comment]
Hi Dean,

You're right, the documentation was wrong in a few places.  I've just updated it now.

Here's what you should see, when this kw is applied:

`_hsr=[dr, 2000, 500], [clr, purple], [cls, ktlFlashingOnOff, ktlOutline], [style, background-color: indianred; font-weight: bold; color: yellow;]`

This gives the following result:

![image](https://github.com/cortexrd/Knack-Toolkit-Library/assets/42851359/32ae1c9d-962a-4e63-b5ce-202df2d6f6a9)


Oops.. I see that the yellow test should be applied to all fields.  I'll look into this later.


Let me know if this works for you,
Norm

**Comment by djbbiz** [comment]
Got it, thanks Norm

**Comment by cortexrd** [comment]
About the wrong color for links (should be yellow), it will be fixed in next release.
Norm
